Assets/PatchKit Patcher/Scripts/AppData/Local/MapHashExtractedFiles.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+namespace PatchKit.Unity.Patcher.AppData.Local
+{
+    public class MapHashExtractedFiles
```

Comment:
Please make it an instance instead of static class and share it between the classes that are using it.

Reply:
It can always be passed along `unarchivePath`.

Reply:
I created  instance instead of static class, but what is going on with unarchivePath?